notebooks/239-image-bind/239-image-bind-quantize.ipynb

```diff
@@ -0,0 +1,805 @@
+{
```

Comment:
<p>on some platforms, there can be difficult to run librosa without soundfile as backend. Could you please add it in requiremnts?</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Yes, I can. I didn't do it before, because soundfile is installed in convert notebook